code/game/gamemodes/cult/runes.dm

```diff
@@ -149,7 +149,10 @@ structure_check() searches for nearby cultist structures required for the invoca
 	if(invocation)
 		for(var/M in invokers)
 			var/mob/living/L = M
-			L.say(invocation)
+			if(!L.IsVocal())
```

Comment:
Any idea why this isn't handled in `say` directly?

Reply:
no idea but do you want me to handle it there or?